packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -92,16 +92,23 @@ pub async fn root_columns_list_by_id(
     #[cfg(feature = "metrics")]
     METRICS.db.postgres.root_columns_list_by_id_calls.inc();
 
-    sqlx::query_as!(
-        RootColumns,
-        r#"SELECT
-               id AS "id: i64", root_id AS "root_id: i64", column_name, graphql_type
-           FROM graph_registry_root_columns
-           WHERE root_id = $1"#,
-        root_id
-    )
-    .fetch_all(conn)
-    .await
+    let rows =
```

Comment:
Just a style nit. This could be chained into one single `Ok`
```suggestion
    Ok(sqlx::query("SELECT * FROM graph_registry_root_columns WHERE root_id = $1")
            .bind(root_id)
            .fetch_all(conn)
            .await?
            .into_inter()
            .map(|row| RootColumns {
              id: row.get("id"),
              root_id: row.get("root_id"),
              column_name: row.get("column_name"),
              graphql_type: row.get("graphql_type"),
        })
        .collect::<Vec<Rootcolumns>>())
            
```

Reply:
For consistency, you can do the individual `row.get(..)`s with the type annotations inside of a map closure and have the closure return a `RootColumns` struct.